study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/RarexVersion3.java

```diff
@@ -0,0 +1,330 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import com.typesafe.config.Config;
+import org.apache.commons.collections.CollectionUtils;
+import org.broadinstitute.ddp.cache.LanguageStore;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.studybuilder.ActivityBuilder;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindList;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Set;
+
+public class RarexVersion3 implements CustomTask {
```

Comment:
Can we execute RarexVersion3 on existing study after RarexVersion2 task is executed ? Looks so but wanted to get it confirmed.

Reply:
Yes, this patch should follow RarexVersion2